ops/ansible/playbooks-healthapt/backend.yml

```diff
@@ -116,6 +116,16 @@
         mode: 'u=rw,g=,o='
       become: true
 
+    # Overwrite the bfd-server role generation of the keystore file with the one in the ansible vault 
+    - name: Copy the appserver keystore  
+      copy:
+        # Note: this JKS file is encrypted via the Ansible Vault 
+        src: "{{ data_server_appserver_keystore }}"
+        dest: '/u01/jboss/bluebutton-appserver-keystore.jks'
+        force: yes # overwrite the existing keystore
+      when: data_server_appserver_keystore is defined
+      become: true
+
```

Comment:
For this to actually take effect, JBoss will have to be restarted.

For that and other reasons, I'd really recommend you move this task inside the `bfd-server` role as an optional parameter.